src/views/VideoMania/index.jsx

```
@@ -111,13 +107,20 @@ const VideoMania = () => {
         ) : topVideo ? (
           <div className="flex justify-center items-center gap-5">
-            <div className="border border-gray-200 p-2 rounded-md">
-              <img src={videoIcon} className="w-32 h-28 overflow-hidden" />
+            <div
+              className="top-video-card"
+              onClick={() => {
+                setCurrentVideo(topVideo);
+                setVideoModal(true);
+                setIsTop(true);
+              }}
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)